### ISSUE TITLE
Font problems: MathBB/Mathcal/Mathfrak

### ISSUE THREAD

**Issue by JorisBelier** [benjaminabel/nodebb-plugin-katex#27]
Hi,

I've a problem with the way some output is displayed. If I use the plugin stand-alone, the code editor will display fonts like mathbb/mathcal/mathfrak in plain font. Then I tried to solve this problem by adding the katex stylesheet to the header in nodebb (The min stylesheet version hosted on their website). This solved the problem of the fonts, however features like subscript/superscript/fractures are not functioning well anymore. 

I added:
````
<!DOCTYPE html>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.8.3/katex.min.css" integrity="sha384-B41nY7vEWuDrE9Mr+J2nBL0Liu+nl/rBXTdpQal730oTHdlrlXHzYMOhDU60cwde" crossorigin="anonymous">
````

to the custom header

I'm a missing some configurations?

I use NodeBB 1.5.2

**Comment by ghost** [comment]
Hi @JorisBelier. Can you provide examples of latex code that do not work please.

**Comment by ghost** [comment]
I've published a new version 0.4.0 with latest katex `lib` and `static` files. This should be fixed, however katex doesn't seem to support all latex styles yet. Can you test it and close the issue if ok.
![special-fonts](https://user-images.githubusercontent.com/1004282/30296790-b6bb25d4-9745-11e7-8764-4fde2161907b.png)

**Comment by JorisBelier** [comment]
Thank you very much, this solved the problem.
